tasks.py

```diff
@@ -282,8 +282,8 @@ def content_excludes():
     return output
 
 
-@task(help={'filename': "Output filename (default = 'data.json')"})
-def export_records(c, filename='data.json'):
+@task(help={'filename': "Output filename (default = 'data.json')", 'overwrite': "Overwrite existing files without asking first (default False)"})
```

Comment:
Nitpik: it should be 'default = False'

Reply:
Nitpik back: why is it default = ... with spaces around the = sign in the help, but without spaces otherwise according to the style check?
None of this is consistent within the file at the moment, by the way.. see import_records() and delete_data() as well.

Reply:
To keep it better readable and it is defined that way on the docstyle we use.
Pep8 is without spaces so the code is that way.